app/src/main/res/values/strings.xml

```diff
@@ -87,4 +92,7 @@
         <item>Every year</item>
         <item>Specific days of the week</item>
     </string-array>
+
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```

Comment:
Hello blank fragment, good bye blank fragment. (Needs to be removed as its not used)